docs/sources/operators-guide/running-production-environment-with-helm/configuration-with-helm.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Manage the configuration of Grafana Mimir with Helm"
+menuTitle: "Manage the configuration of Grafana Mimir with Helm"
+description: "Learn how to customize, secure and update the Grafana Mimir configuration using the Helm chart."
+weight: 80
+---
+
+# Manage the configuration of Grafana Mimir with Helm
+
+The `mimir-distributed` Helm chart provides a single interface to set Grafana Mimir [configuration parameters]({{< relref "../configure/reference-configuration-parameters/" >}}) and customize how Grafana Mimir is deployed on a Kubernetes cluster. This document is about the configuration parameters.
+
+## Overview
+
+The Grafana Mimir configuration can be managed through the Helm chart or supplied via a user managed object.
+
+If you want to manage the configuration via the Helm chart, see [Manage the configuration with Helm](#manage-the-configuration-with-helm).
+
+If you want to manage the configuration directly, see [Manage the configuration directly](#manage-the-configuration-directly).
+
+Handling sensitive information, such as credentials is common between the two methods, see [Injecting credentials](#injecting-credentials).
+
+## Manage the configuration with Helm
+
+There are three ways configuration parameters can be modified:
+
+1. Copying the whole `mimir.config` value and modifying the configuration as text
+1. Setting parameters via the `mimir.structuredConfig` value (recommended)
```

Comment:
nit: can you put the recommended way as the first option?
```suggestion
1. Setting parameters via the `mimir.structuredConfig` value (recommended)
1. Copying the whole `mimir.config` value and modifying the configuration as text
```

Reply:
I would even go further and add `(discouraged)` to the `mimir.config` version.

Reply:
reordered and added suggestions, please review